packages/dripsy/package.json

```diff
@@ -75,18 +75,8 @@
       "typescript"
     ]
   },
-  "peerDependencies": {
-    "react": "16.13.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz"
-  },
   "dependencies": {
-    "@artsy/fresnel": "^1.2.0",
-    "@expo/html-elements": "^0.0.1",
-    "@react-native-community/hooks": "^2.6.0",
-    "@theme-ui/core": "^0.4.0-rc.1",
-    "@theme-ui/css": "^0.4.0-rc.1",
-    "hoist-non-react-statics": "^3.3.2",
-    "theme-ui": "^0.4.0-rc.3"
+    "@dripsy/core": "*"
```

Comment:
@cmaycumber what's best practice for this? I'm unaware. Does the * just allow any version? Presumably we'd want the versions to match up. I saw lerna has an `exact` option, not sure if that's what we want or not.

Reply:
Possibly related:

Reply:
Since `lerna publish` will make all the version increment, I wonder if there's a way to make them also increment in the respective `package.json` files.

Reply:
I notice here that they use exact versions: https://github.com/system-ui/theme-ui/blob/ca00080aa3ceee704ad1fc4de99ada044e9b9193/packages/theme-ui/package.json#L22

I'm also wondering what the answer is for moti's sake.

Reply:
> @cmaycumber what's best practice for this? I'm unaware. Does the * just allow any version? Presumably we'd want the versions to match up. I saw lerna has an `exact` option, not sure if that's what we want or not.

Yeah, it's just the latest. I've looked around for something that manages exact versions in a monorepo and haven't been able to find anything that locks versions/updates them across packages. 

We can switch to the exact versions I'm curious if there's a "best practice" solution for incrementing the versions or if we'll just have to write a script.

Reply:
So it looks like lerna does this for you if you use semver versioning  `^0.x.y`.  I added versions like that in Moti, and when I ran `lerna publish`, it incremented all the dependencies for me.